packages/tools/src/drawingSvg/drawLine.ts

```diff
@@ -11,34 +11,50 @@ export default function drawLine(
   lineUID: string,
   start: Types.Point2,
   end: Types.Point2,
-  options = {},
+  options: {
+    color?: string;
+    width?: number;
+    lineWidth?: number;
+    lineDash?: string;
+    markerStartId?: string;
+    markerEndId?: string;
+    shadow?: boolean;
+  },
   dataId = ''
 ): void {
   // if length is NaN return
   if (isNaN(start[0]) || isNaN(start[1]) || isNaN(end[0]) || isNaN(end[1])) {
     return;
   }
 
-  const { color, width, lineWidth, lineDash, shadow } = Object.assign(
-    {
-      color: 'rgb(0, 255, 0)',
-      width: '2',
-      lineWidth: undefined,
-      lineDash: undefined,
-      shadow: undefined,
-    },
-    options
-  );
+  const {
+    color = 'rgb(0, 255, 0)',
+    width = 10,
+    lineWidth,
+    lineDash,
+    markerStartId = null,
+    markerEndId = null,
+    shadow = false,
+  } = options;
 
   // for supporting both lineWidth and width options
   const strokeWidth = lineWidth || width;
 
   const svgns = 'http://www.w3.org/2000/svg';
   const svgNodeHash = _getHash(annotationUID, 'line', lineUID);
   const existingLine = svgDrawingHelper.getSvgNode(svgNodeHash);
-  const dropShadowStyle = shadow
-    ? `filter:url(#shadow-${svgDrawingHelper.svgLayerElement.id});`
-    : '';
+  const layerId = svgDrawingHelper.svgLayerElement.id;
+  const dropShadowStyle = shadow ? `filter:url(#shadow-${layerId});` : '';
+  const arrow = svgDrawingHelper.svgLayerElement.querySelector(
+    `#${markerEndId}-${layerId}`
+  );
+
+  if (arrow) {
+    arrow.setAttribute('fill', color);
+    const newstrokeWidth = strokeWidth >= 4 ? 3 : 6;
```

Comment:
Why is `strokeWidth >= 4 ? 3 : 6;` used here? It seems arbitrary.

Reply:
if the stroke is more than 4 the arrow becomes bigger hence the reduction to 3 otherwise it's 6 but i think i need to find a better way for this

Reply:
I guess put it in the configuration